src/graphql/root/mutation/user-request-auth-code.ts

```diff
@@ -29,10 +29,9 @@ const UserRequestAuthCodeMutation = GT.Field({
       return { errors: [{ message: phone.message }] }
     }
 
-    // TODO: Make this through a new app use-case
     const status = await requestPhoneCode({ phone, logger, ip })
 
-    if (!status) {
+    if (status instanceof Error) {
```

Comment:
now we can use map error to show the proper error message `"Could not complete the operation successfully"` should be a message from error mapping